homeassistant/components/wiz/__init__.py

```diff
@@ -80,10 +82,14 @@ async def async_setup_entry(hass: HomeAssistant, entry: ConfigEntry) -> bool:
             "Found bulb {bulb.mac} at {ip_address}, expected {entry.unique_id}"
         )
 
-    async def _async_update() -> None:
+    async def _async_update() -> float | None:
         """Update the WiZ device."""
         try:
             await bulb.updateState()
+            if bulb.power_monitoring is not False:
```

Comment:
`is not False`? can we just do
```suggestion
            if bulb.power_monitoring:
```

Reply:
It could be `None` which means it is not initialized yet so we have to poll to get it initialized. `False` means its tried to do it and failed so the device doesn't support it and we shouldn't try again.